tools/fds/server.h

```diff
@@ -97,7 +105,15 @@ const option::Descriptor usage[] = {
       "\t5. Launch a default server with id 0 (first on ROS_DISCOVERY_SERVER)\n"
       "\t   listening on localhost with UDP port 14520. Only localhost clients\n"
       "\t   can reach the server defining as ROS_DISCOVERY_SERVER=localhost:14520.\n\n"
-      "\t$ " FAST_SERVER_BINARY " -i 0 -l localhost -p 14520"},
+      "\t$ " FAST_SERVER_BINARY " -i 0 -l localhost -p 14520\n\n"
+
+      "\t6. Launch a server with id 0 (first on ROS_DISCOVERY_SERVER) reading\n"
+      "\t   default configuration from XML file.\n\n"
+      "\t$ " FAST_SERVER_BINARY " -i 0 -x config.xml\n\n"
+
+      "\t6. Launch a server with id 0 (first on ROS_DISCOVERY_SERVER) reading\n"
+      "\t   specific profile_name configuration from XML file.\n\n"
+      "\t$ " FAST_SERVER_BINARY " -i 0 -x config.xml@profile_name"},
```

Comment:
If we change the profile position, this should be changed

Reply:
Done in 6dd1ad92f